ligo/src/registryDAO/types.mligo

```diff
@@ -10,3 +10,13 @@ type registry_value = string
 type registry = (registry_key, registry_value) map
 type registry_affected = (registry_key, proposal_key) map
 type registry_diff = (registry_key * registry_value option) list
+type proposal_receivers = address set
+
+type update_receiver_param =
+  | Add_receivers of (address list)
+  | Remove_receivers of (address list)
```

Comment:
Looking at the comment in the issue it seems this might be incorrect?

The request (in case of a single proposal) is to have something like:
> One proposal type: receiver_diff : pair (list %remove address) (list %add address)

which sounds like it should be able to support adding and removing in the same proposal.

Although maybe that's not the case, given that if we went with two proposals only one thing at a time would be possible, To be sure I think this should be asked.

Reply:
>which sounds like it should be able to support adding and removing in the same proposal.

I am not sure, but It seems that allowing that will result in some ambiguity w.r.t to the order of operations when the both addition and removal lists contain the same address, because right now adding existing items, and removing non-existing ones are no-ops. 

So for every duplicate item in the lists, If we apply addition before removal, then existing copies will be removed. If we apply removal before addition, then the new instance will be added.

We can do some processign or validation on the input, but that will probably increase the size which AFAIU is not desireable.

Reply:
That's a good point, the outcome would be more problematic :ok_hand: